cli/ops/process.rs

```diff
@@ -22,19 +22,11 @@ pub fn init(i: &mut Isolate, s: &State) {
 
 fn clone_file(rid: u32, state: &State) -> Result<std::fs::File, OpError> {
   let mut state = state.borrow_mut();
-  let repr_holder = state
-    .resource_table
-    .get_mut::<StreamResourceHolder>(rid)
-    .ok_or_else(OpError::bad_resource_id)?;
-  match repr_holder.resource {
-    StreamResource::FsFile(Some((ref mut file, _))) => {
-      let tokio_file = futures::executor::block_on(file.try_clone())?;
-      let std_file = futures::executor::block_on(tokio_file.into_std());
-      Ok(std_file)
-    }
-    StreamResource::FsFile(None) => Err(OpError::resource_unavailable()),
-    _ => Err(OpError::bad_resource_id()),
-  }
+
+  std_file_resource(&mut state.resource_table, rid, move |r| match r {
+    Ok(std_file) => std_file.try_clone().map_err(OpError::from),
+    Err(_) => Err(OpError::bad_resource_id()),
+  })
 }
```

Comment:
If one clones a file, should there be a new entry in the resource table?

Reply:
No - it's same underlying file - we clone just because of API contract that requires `Into<Stdio>` which requires owned file.

Reply:
Hm. I'm not sure about this, but this is probably better than before. Let's land it for now and I'll revisit later.